stairway/src/main/java/bio/terra/stairway/StairwayBuilder.java

```diff
@@ -30,7 +32,7 @@ public class StairwayBuilder {
 
   /**
    * Determines the size of the thread pool used for running Stairway flights. Default is
-   * DEFAULT_MAX_PARALLEL_FLIGHTS (20 at this moment)
+   * established in {@link DefaultThreadPoolTaskExecutor} (20 at this moment).
```

Comment:
Could just ponit to 
```suggestion
   * established in {@link DefaultThreadPoolTaskExecutor.DEFAULT_MAX_PARALLEL_FLIGHTS}
```

And I don't know that I'd put the value since that could get out of sync

Reply:
I had tried the link like this and it didn't work, but I figured out how to get it to work… much better :)

<img width="577" alt="Screenshot 2024-07-01 at 3 10 07 PM" src="https://github.com/DataBiosphere/stairway/assets/79769153/d16f611b-6bdb-40c2-874b-0f7a30e78ae5">
<img width="728" alt="Screenshot 2024-07-01 at 3 10 15 PM" src="https://github.com/DataBiosphere/stairway/assets/79769153/b3caf76c-8d7c-48ed-8530-86291d2a677c">

Reply:
Ah good find!